src/Components/Activities/Table/table.module.css

```diff
@@ -0,0 +1,38 @@
+table {
```

Comment:
Try to avoid using universal selectors, as they will override styles for all elements